Assets/LeapMotion/Core/Scripts/DataStructures/Pose.cs

```diff
@@ -0,0 +1,167 @@
+﻿using UnityEngine;
+
+namespace Leap.Unity {
+
+  /// <summary>
+  /// A position and rotation.
+  /// </summary>
+  [System.Serializable]
+  public struct Pose {
+
+    public Vector3    position;
+    public Quaternion rotation;
+
+    public Pose(Vector3 position, Quaternion rotation) {
+      this.position = position;
+      this.rotation = rotation;
+    }
+
+    public static Pose identity {
+      get { return new Pose(); }
```

Comment:
Watch out, when a struct is instantiated, all fields are initialized to their default values.  So the rotation is initialized to the default value of Quaternion.  Quaternion is also a struct, so the same rule applies and all of it's fields (x, y, z, _and_ w) get initialized to zero, making the quaternion invalid instead of the identity quaternion.

I'd recommend something along the lines of 
`public static readonly Pose identity = new Pose(Vector3.zero, Quaternion.identity);`

Reply:
oh geez, good catch

Reply:
done